src/main/java/com/airbnb/billow/AWSDatabase.java

```diff
@@ -79,6 +77,37 @@
         this.ec2SGs = ec2SGbuilder.build();
 
         /*
+         * IAM keys
+         */
+
+        log.info("Getting IAM keys");
+        String accountNumber = ""; // we need this to describe RDS
+        final ImmutableList.Builder<IAMUserWithKeys> usersBuilder = new ImmutableList.Builder<IAMUserWithKeys>();
+
+        final ListUsersRequest listUsersRequest = new ListUsersRequest();
+        ListUsersResult listUsersResult;
+        do {
+            log.debug("Performing IAM request: {}", listUsersRequest);
+            listUsersResult = iamClient.listUsers(listUsersRequest);
+            final List<User> users = listUsersResult.getUsers();
+            log.debug("Found {} users", users.size());
+            for (User user : users) {
```

Comment:
Then account number could not be identified and describing RDS will fail. Since we'll most likely always have many users under the account, can we push this to future improvement? I want to push back adding a *mandatory* `accountNumber` arg until it's really necessary.

An alternative to infer accountNumber without passing in is to try to describe a resource you don't have access to. In that case you'll get `Access Denied` and your account number will be in the 403 error message. But you'll have to work with permission setup to make sure you'll get `Access Denied` on that resource.